cmd/beyondctl/main.go

```diff
@@ -38,3 +39,11 @@ func main() {
 		logger.Fatal("beyondctl execute", zap.Error(err))
 	}
 }
+
+func userConfigDir() string {
+	configDir, err := os.UserConfigDir()
```

Comment:
Why not user home dir?

Reply:
Because we already have config dir here, we just need to follow the rule.

Reply:
Use home dir directly is not the best practice. In the Linux platform, we have `XDF_CONFIG_DIR` to set the user's config directory.

Reply:
See the comment of `os.UserConfigDir` for more details.